### DISCUSSION TITLE
Brew commands just hang in terminal

### DISCUSSION THREAD

**Discussion by MincePie** [Homebrew/discussions#3192] [Everyday usage]
I just updated to macOS Monterey 12.3.1.

When I try to use brew commands in my terminal, the system just hangs. For example, brew search node, and brew doctor, just sit until I control C to terminate the process. I have left each of these running all day.

Since upgrading, my mac has had a window that says 'verifying xcode'. It takes an age to run, and even if I press the x to close the process, the next time I start my mac, it is trying to run again. 

Is brew now incompatible with macOS? Is there a way to fix this?

**Comment by Bo98** [comment]
> Since upgrading, my mac has had a window that says 'verifying xcode'.

Given some macOS tools like `git` go through Xcode, this is likely the cause.

Have you let it run to completion?

You can also try:
```
xattr -d com.apple.quarantine /Applications/Xcode.app
```
to skip it, assuming you're sure you downloaded from a trusted source of course.
